### PR TITLE
Feature/#438 handle status updates (part 1)

### PR DESCRIPTION
**Pull request by das-g**
First part of #438:

### Implemented in this PR:
- If callback received and status in callback payload different from export model instance status:
  1. update export model instance status
  2. notify user by on-screen message
- Share status definitions between conversion service and frontend

### Still to do in subsequent PRs:
- Query conversion service for jobs of state export statuses
- Decide in more detail what should happen upon what status change (see https://github.com/geometalab/osmaxx-frontend/issues/438#issue-148772137) and implement that accordingly.
- Remove status (field `state`) from extraction order model
- Move downloads from extraction order model to export model

### Reviewed by
- [x] @hixi

### Please note
- Export model statuses are initialized/set independent of extraction order status. So the information about the order's states will be lost when the corresponding field is removed in favor of the new field on the export model.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def pytest_configure():
             'rest_framework_gis',
             'rest_framework.authtoken',
             'crispy_forms',
+            'stored_messages',
```
Comment (hixi) [inline tests/conftest.py]: Are we relying in tests on this app being present? I couldn't find it in the changed code, but that doesn't mean it's not used ;-)

**Comment by hixi** [comment]
very nice! :+1:

File: tests/conftest.py (hunk 1 above)
Comment (das-g) [inline tests/conftest.py]: > Are we relying in tests on this app being present? I couldn't find it in the changed code [...]

That would be [the call to](https://github.com/geometalab/osmaxx-frontend/pull/446/files#diff-e52c21d719a322835dc4db89ea595076R58) `osmaxx.utilities.shortcuts.Emissary.info` which uses `stored_messages` [via `Emissary.inform`](https://github.com/geometalab/osmaxx-frontend/blob/0704004e8da31444f947bec022ef60132284224d/osmaxx/utilities/shortcuts.py#L80).

Comment (hixi) [inline tests/conftest.py]: :+1:

**Pull request merged**